controller/workitem_blackbox_test.go

```diff
@@ -320,7 +320,7 @@ func createPagingTest(t *testing.T, controller *WorkitemController, repo *testsu
 		count := computeCount(totalCount, int(start), int(limit))
 		repo.ListReturns(makeWorkItems(count), uint64(totalCount), nil)
 		offset := strconv.Itoa(start)
-		_, response := test.ListWorkitemOK(t, context.Background(), nil, controller, nil, nil, nil, nil, nil, &limit, &offset)
+		_, response := test.ListWorkitemOK(t, context.Background(), nil, controller, nil, nil, nil, nil, nil, nil, &limit, &offset)
```

Comment:
Suggestion (not a blocker): We could improve these function calls, if you move limit and offset params before the nil chain of params. Then you can simply call the function omitting the nil params 
`test.ListWorkitemOK(t, context.Background(), nil, controller, &limit, &offset)`